index.js

```diff
@@ -62,24 +62,16 @@ module.exports = function (options) {
 			if (errorList.length > 0) {
 				file.jscs.success = false;
 				file.jscs.errorCount = errorList.length;
-				file.jscs.errors = errorList;
+				file.jscs.errors = errors;
 			}
-
-			errorList.forEach(function (err) {
-				out.push(errors.explainError(err, true));
-			});
 		} catch (err) {
-			out.push(err.stack.replace('null:', file.relative + ':'));
+			console.error(err.stack.replace('null:', file.relative + ':'));
```

Comment:
I've been feeding all kinds of gibbering to gulp-jscs and at most it reports "Unexpected token", so I believe we can remove this try/catch.